### DISCUSSION TITLE
用latexmk-sjtuthesis配方编译失败

### DISCUSSION THREAD

**Discussion by lcq-molybdenum42** [sjtug/SJTUThesis#1074] [提问]
前期一直用知乎上搜来的workshop配置，中西文混排的空格之类问题令人抓狂，后面发现是没有用xelatex导致的（原有配置似乎是pdflatex），于是按照使用说明，配置了latexmk-sjtuthesis配方，但是却在编译时直接报错。信息如图所示。
![image](https://github.com/user-attachments/assets/aa519d92-0339-4be3-94e5-4cff6cab49fc)

**Comment by LogCreative** [comment]
感觉可能你的发行版环境出问题了？可以先在命令行使用
```bash
fc-cache
```
试试

**Reply by lcq-molybdenum42** [reply]
![image](https://github.com/user-attachments/assets/d76f585a-495d-4f73-9102-f5e5deb14777)
yun运行这一行时出现了相同的报错。。。

> 可能你的发行版环境出问题了

如何判断我的环境有没有问题呢？

**Reply by LogCreative** [reply]
Cannot load default config file: No such file: (null)

建议重装 TeX Live 2024 可能有些东西被清理了
